pkg/runtime/interfaces/application_configuration.go

```diff
@@ -17,8 +18,11 @@ type DbConfig struct {
 	DeprecatedExtraOptions string `json:"options" pflag:",deprecated"`
 	DeprecatedDebug        bool   `json:"debug" pflag:",deprecated"`
 
-	EnableForeignKeyConstraintWhenMigrating bool           `json:"enableForeignKeyConstraintWhenMigrating" pflag:",Whether to enable gorm foreign keys when migrating the db"`
-	PostgresConfig                          PostgresConfig `json:"postgres"`
+	EnableForeignKeyConstraintWhenMigrating bool            `json:"enableForeignKeyConstraintWhenMigrating" pflag:",Whether to enable gorm foreign keys when migrating the db"`
+	MaxIdleConnections                      int             `json:"maxIdleConnections" pflag:",maxIdleConns sets the maximum number of connections in the idle connection pool."`
+	MaxOpenConnections                      int             `json:"maxOpenConnections" pflag:",maxOpenConns sets the maximum number of open connections to the database."`
```

Comment:
I would use the full name in the description to avoid confusion, i.e. maxIdleConnections instead of maxIdleConns and maxOpenConnections instead of maxOpenConns. For example: "maxIdleConnections sets the maximum number of connections in the idle connection pool."

Reply:
Done